replicator/run.py

```diff
@@ -112,6 +112,7 @@ def parse(self):
 
             members.add(name)
 
+        assert members
```

Comment:
empty members will now crash the replicator. Is that intentional?

Reply:
Yes. Before, if `align-catalog-zone` would run at the stack (when starting the docker-compose application), the catalog zone would be removed and recreated at the stack. For a short time, the zone would then exist but be empty; during that time, slaves would start removing all zones.

It is much better to crash and restart the `replicator` container in this case.

Reply:
Indeed. `assert` is fine or did you want to raise a proper exception?

Reply:
done